Source/Router.swift

```diff
@@ -16,13 +16,14 @@
 
 import UIKit
 
-public class Router {
+typealias RouteHandler = (req: Request) -> Void
+
+class Router {
     
-    public typealias RouteHandler = (req: Request) -> Void
-    private var orderedRoutes: [Route] = [Route]()
-    private var routes: [Route: RouteHandler] = [Route: RouteHandler]()
+    private var orderedRoutes = [Route]()
+    private var routes = [Route: RouteHandler]()
     
-    public init() {}
+    init() {}
```

Comment:
this actually needs to be public init() {} to expose a constructor to the example app